lib/pure/oids.nim

```diff
@@ -44,14 +44,15 @@ proc hexbyte*(hex: char): int =
   else: discard
 
 proc parseOid*(str: cstring): Oid =
-  ## parses an OID.
+  ## Parses an OID.
   var bytes = cast[cstring](addr(result.time))
   var i = 0
   while i < 12:
     bytes[i] = chr((hexbyte(str[2 * i]) shl 4) or hexbyte(str[2 * i + 1]))
     inc(i)
 
 proc oidToString*(oid: Oid, str: cstring) =
+  ## Converts a oid to `str`.
```

Comment:
in a future PR I think we should deprecate `oidToString`.
it's only used in 
```
proc `$`*(oid: Oid): string =
  result = newString(24)
  oidToString(oid, result)
```
and there are better/safer ways to do this

Reply:
sorry, I submit wrong commits. I will revert this
https://github.com/nim-lang/Nim/pull/16203